heartbeat/monitors/active/http/task.go

```diff
@@ -218,41 +218,62 @@ func execPing(
 	body []byte,
 	timeout time.Duration,
 	validator func(*http.Response) error,
-) (time.Time, time.Time, common.MapStr, reason.Reason) {
+) (start time.Time, end time.Time, event common.MapStr, errReason reason.Reason) {
```

Comment:
can also be written `(start, end time.Time, event common.MapStr, err reason.Reason)`